tests/peer/test_peerid.py

```diff
@@ -1,9 +1,117 @@
+import random
+import multihash
+import pytest
+import base58
 from Crypto.PublicKey import RSA
-from peer.id import id_from_private_key, id_from_public_key
+from peer.id import ID, id_b58_encode, id_b58_decode, id_from_public_key, id_from_private_key
 
+ALPHABETS = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
+
+def test_init_():
+    random_id_string = ''
+    for _ in range(10):
+        random_id_string += random.SystemRandom().choice(ALPHABETS)
```

Comment:
We should **de-randomize** this test and the others. Though it might make the tests slightly less comprehensive over several runs, we can't have variability in the results since it'll make it harder to evaluate new changes. Maybe we can test with a few starter "seed" strings.

Reply:
I do not understand what you meant